extras/shader-nodes/add.js

```diff
@@ -0,0 +1,17 @@
+import { flexOp } from "./flexOp";
```

Comment:
for testing it's ok, but don't add them to /extras directly, create a new folder for these and add build step similar to extras to build them into separate js file.

Reply:
see the end of rollup.config.js

Reply:
Done (adding to it extras was just for testing)